File: BE/health-service/src/main/java/com/s206/health/nutrition/meal/dto/response/DailyNutritionResponse.java
package com.s206.health.nutrition.meal.dto.response;

import lombok.Builder;
import lombok.Getter;

import java.math.BigDecimal;
import java.time.LocalDate;

@Getter
@Builder
public class DailyNutritionResponse {

    private LocalDate date;

    private Integer totalCalorie;

    private BigDecimal totalCarbohydrate;

    private BigDecimal totalProtein;

    private BigDecimal totalFat;

    private BigDecimal totalSugar;

    private BigDecimal totalSodium;
}

File: FE/mobile/src/main/java/com/example/diaviseo/network/meal/dto/res/DailyNutritionResponse.kt
package com.example.diaviseo.network.meal.dto.res

import java.math.BigDecimal
import java.time.LocalDate

data class DailyNutritionResponse(
    val date: String,
    val totalCalorie: Int,
    val totalCarbohydrate: Double,
    val totalProtein: Double,
    val totalFat: Double,
    val totalSugar: Double,
    val totalSodium: Double
)